### PR TITLE
feat(git-practice): Tamara's rehearsal

### PR DESCRIPTION
**Pull request by carrizotamara**
attached files html

### CODE REVIEW

**Comment by arielarmijo** [comment]
well done!

**Review by arielarmijo — DISMISSED** [review]

**Review by arielarmijo — DISMISSED** [review]
Well done

**Review by RickyCode — COMMENTED** [review]
please delete the .jpg and .png files

**Review by RickyCode — APPROVED** [review]

**Review by arielarmijo — APPROVED** [review]
well done

**Comment by RickyCode** [comment]
great job!

**Review by d-jv — APPROVED** [review]
Looks fine to me.👍

File: tamara-git-practice/assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+section, footer {
+  margin-top: 4.5rem;
+}
+
+footer {
+  padding: 3.125rem;
+}
+
+.centrado {
+  width: 75%;
```
Comment (mframirezfuentes) [inline tamara-git-practice/assets/css/style.css:44]: change percent

Comment (carrizotamara) [reply tamara-git-practice/assets/css/style.css:44]: ok, thanks i will make the modification

**Pull request merged**